recipe/meta.yaml

```diff
@@ -19,30 +19,32 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.9
     - pip
 
   run:
-    - python >=3.8
+    - python >=3.9
     - pip
     - beautifulsoup4
-    - cartopy
+    - cartopy >=0.17.0
     - cartopy_offlinedata
-    - cdp
-    - cdms2
-    - cdutil
-    - genutil
+    - cdp 1.7.0
+    - cdms2 3.1.5
+    - cdutil 8.2.1
+    - dask
+    - genutil 8.2.1
     - lxml
-    - mache
+    - mache >=0.15.0
     - matplotlib-base
     - netcdf4
-    - numpy
-    - xarray
+    - numpy >=1.23.0
+    - xarray >=2023.02.0
```

Comment:
Updated to match [E3SM-Project/e3sm_diags@d0aa16b/conda-env/ci.yml#L9C1-L26](https://github.com/E3SM-Project/e3sm_diags/blob/d0aa16b099db10159d0b585b24f8c641c5bb780c/conda-env/ci.yml#L9C1-L26)